lib/rack/test/cookie_jar.rb

```diff
@@ -165,15 +165,15 @@ def []=(name, value)
       # no such cookie exists.
       def get_cookie(name)
         @cookies.each do |cookie|
-          return cookie if cookie.name == name
+          return cookie if cookie.name == name.to_s
```

Comment:
I think this change makes sense but we should do the conversion outside of the loop like we do in `#[]`